Audio-Transcription-Firefox/README.md

```diff
@@ -0,0 +1,33 @@
+# Audio Transcription Fox
```

Comment:
Is this the name of the Extension `Fox`? Or should this be `Audio Transcription Firefox`?

Reply:
typo, I had the initial name of the folder as Audio Transcription Fox.